src/backend/distributed/commands/table.c

```diff
@@ -3403,3 +3405,44 @@ MakeNameListFromRangeVar(const RangeVar *rel)
 		return list_make1(makeString(rel->relname));
 	}
 }
+
+
+/*
+ * WarnUnsupportedIfForeignDistributedTable gets a relationId and logs a WARNING
+ * if the given relation is a distributed foreign table.
+ * We do that because now we only support Citus Local Tables for foreign tables.
+ */
+void
+WarnUnsupportedIfForeignDistributedTable(Oid relationId)
```

Comment:
do we really need this function now? Can't we inline into `WarnIfListHasForeignDistributedTable`?  It seems confusing to have an external function with this name. 

And, when we do that, we should probably be able to remove `DistributedForeignTableWarningPrompted` check from this part?